ui/src/dashboards/components/CellEditorOverlay.js

```diff
@@ -73,6 +92,24 @@ class CellEditorOverlay extends Component {
     }
   }
 
+  handleSetRange(e) {
+    const {min, max} = e.target.form
```

Comment:
Please see ![the React docs for controlled components](https://facebook.github.io/react/docs/forms.html).  I think you will be pleasantly surprised how controlled these components are 😸